website/src/components/ReviewPage/ReviewCard.tsx

```diff
@@ -201,20 +205,27 @@ const MetadataList: FC<MetadataListProps> = ({ data, isLoading, metadataDisplayN
 type ErrorsProps = {
     errors: ProcessingAnnotation[];
     accession: string;
+    metadataDisplayNames: Map<string, string>;
 };
 
-const Errors: FC<ErrorsProps> = ({ errors, accession }) => {
+const Errors: FC<ErrorsProps> = ({ errors, accession, metadataDisplayNames }) => {
     return (
         <div>
             <div className='flex flex-col m-2 '>
                 {errors.map((error) => {
-                    const uniqueKey = error.source.map((source) => source.type + source.name).join('.') + accession;
+                    const uniqueKey =
+                        error.processedFields.map((field) => field.type + field.name).join('.') + accession;
+                    const processedFieldName = error.processedFields
+                        .map((field) => metadataDisplayNames.get(field.name) ?? field.name)
+                        .join(', ');
+                    const unprocessedFieldName = error.unprocessedFields.map((field) => field.name).join(', ');
                     return (
                         <div key={uniqueKey} className='flex flex-shrink-0'>
                             <p
                                 className='text-red-600'
                                 data-tooltip-id={'error-tooltip-' + accession + '-' + uniqueKey}
                             >
+                                {processedFieldName} (produced from input fields: {unprocessedFieldName}):{' '}
```

Comment:
<img width="1005" alt="image" src="https://github.com/user-attachments/assets/e3b1b6cf-4cc4-4224-8a97-15ddb9a423ed">

I don't think this is great - I would leave it as is and if we want to rethink it later we can

Reply:
So just get rid of the (produced from ... section)? I do think it is nice to map to the display Name in the warning - otherwise it is confusing as the error is about the geoLocCountry and you see Collection country in the details

Reply:
I think this UI is worse than previously atm. E.g. `main (produced from input fields: main)` when main isn't actually field is one example, but I think it applies to all of them here really. I agree we can put some effort into making this better at some point. (TBH I think one might do it by making the submission page allow you to upload stuff with display names as the column headers, as part of https://github.com/loculus-project/loculus/issues/3399) but I don't think we need to block on this for this PR.

Reply:
(So yes, my suggestion was just getting rid for now)

Reply:
True - it does look very odd for the sequence errors - I have now removed :-)